### PR TITLE
Proposal for fixing long text in paging toolbar

### PR DESCRIPTION
**Pull request by muros-ct**
This is possible solution to paging toolbar, where user can enter arbitrarily long
text for page number.

We already have KapuaPagingToolbar where I tested it and it works.

This fixes issue #1225.

Signed-off-by: Uros Mesaric Kunst <uros.mesaric-kunst@comtrade.com>

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
I have verified that this PR successfully resolves issue #1225.

File: console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/widget/KapuaPagingToolBar.java [hunk 1]
```diff
@@ -19,8 +19,17 @@
 
     private static final ConsoleMessages MSGS = GWT.create(ConsoleMessages.class);
 
+    /**
+     * Max length of user input field for page number.
+     */
+    public static final String MAX_TEXT_LEN = "4";
```
Comment (lorthirk) [inline console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/widget/KapuaPagingToolBar.java]: Wouldn't it be safer to use an `Integer` here, and use its `toString()` in the `setAttibute()` later on?

Comment (muros-ct) [reply console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/widget/KapuaPagingToolBar.java]: Shure, I didn't know from which side to go.

Comment (lorthirk) [reply console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/widget/KapuaPagingToolBar.java]: Well, at least using `Integer` you can be certain that only an integer will be written in the variable :smile:

**Comment by muros-ct** [comment]
@lorthirk I changed to integer.

**Pull request merged**